app/controllers/websites_controller.rb

```diff
@@ -7,25 +7,23 @@ def index
 
   def update
     @website.update_attributes(website_params)
-    redirect_to installations_url, notice: 'Website updated'
+    redirect_to installations_url(details_tab: 'settings', main_tab: 'details'), notice: 'Website updated'
   end
 
   def destroy
     return false unless WebsiteMember.where(user_id: current_user.id, website_id: @website.id).first.role == "owner"
     @website.destroy
-    respond_to do |format|
-      unless current_user.websites.blank?
-        set_website(current_user.websites.first)
-        format.js { render inline: 'location.reload();' }
-      end
-      format.js { render inline: "location.href='#{website_wizard_path(:create)}';" } if current_user.websites.blank?
+    unless current_user.websites.blank?
+      set_website(current_user.websites.first)
+      redirect_to websites_url
     end
+    website_wizard_url(:create) if current_user.websites.blank?
```

Comment:
Do a simple `if .. else .. end` block here instead of doing unless and then another if.